cli: skip TestUnavailableZip

otan opened the pull request:
Refs: #53306

Reason: flaky test

Generated by bin/skip-test.

Release note: None

Release justification: non-production code changes

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/53447)
<!-- Reviewable:end -->

otan committed:
cli: skip TestUnavailableZip

Refs: #53306

Reason: flaky test

Generated by bin/skip-test.

Release note: None

Release justification: non-production code changes

otan commented:
bors r=irfansharif

otan commented:
bors r=irfansharif

On Tue, 25 Aug 2020, 5:11 pm craig[bot], <notifications@github.com> wrote:

> Build failed:
>
>    - GitHub CI (Cockroach)
>    <https://teamcity.cockroachdb.com/viewLog.html?buildId=2222680&buildTypeId=Cockroach_UnitTests>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/53447#issuecomment-680333555>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA32FQ4B2C75FSF2UIYRGIDSCRHJRANCNFSM4QLF2KTA>
> .
>

otan commented:
bors r=irfansharif
